reader-sdk-react-native-quickstart/android/app/build.gradle

```
@@ -201,24 +204,28 @@ android {
 
         }
     }
+
+    lintOptions {
+        checkReleaseBuilds false
+        abortOnError false
```

Why do we ignore errors?

---

Removed !!